### PR TITLE
Supersede gql

### PR DESCRIPTION
**Pull request by LegoStormtroopr**
Issues resolved: #

Sanity checks:
- [ ] Have tests been run locally?
- [ ] Does this change any javascript? If so, have you checked that the UI works as expected?
- [ ] If there are data model changes, are relevant data standards referenced - in code?
- [ ] If there are deviations from any specified standards, are the reasons for changes documented?
- [ ] If there are UI changes have you documented whats changed so the help documentation can be updated?

### CODE REVIEW

File: python/aristotle-mdr-graphql/aristotle_mdr_graphql/aristotle_filterset_classes.py [hunk 1]
```diff
@@ -56,3 +56,16 @@ def filter_only_public(self, qs, name, value):
             return qs.public()
         else:
             return qs
+
+
+class SupersedeRelationshipFilterSet(FilterSet):
+    approved = django_filters.BooleanFilter(method='filter_approved')
```
Comment (harrywhite4) [inline python/aristotle-mdr-graphql/aristotle_mdr_graphql/aristotle_filterset_classes.py]: Opposite of proposed, should be commented

File: python/aristotle-metadata-registry/aristotle_mdr/managers.py [hunk 2]
```diff
@@ -446,16 +446,42 @@ def current(self, when=timezone.now()):
         return states.select_related('registrationAuthority')
 
 
-class SupersedesManager(models.Manager):
+class SupersedesQueryset(models.QuerySet):
+    def visible(self, user):
+        from aristotle_mdr.models import _concept
+        visible_concepts = _concept.objects.visible(user)
+        editable_concepts = _concept.objects.visible(user)
```
Comment (harrywhite4) [inline python/aristotle-metadata-registry/aristotle_mdr/managers.py]: visible and editable the same, function should be refactored

File: python/aristotle-metadata-registry/aristotle_mdr/managers.py [hunk 3]
```diff
@@ -446,16 +446,42 @@ def current(self, when=timezone.now()):
...
+            return self.filter(
+                newer_item__in=visible_concepts,
+                older_item__in=visible_concepts,
+                proposed=False
+            )
+
+        if user.is_superuser:
+            return self.all()
 
+        return self.filter(
```
Comment (harrywhite4) [inline python/aristotle-metadata-registry/aristotle_mdr/managers.py]: Need to decide on the appropriate permission here

File: python/aristotle-metadata-registry/aristotle_mdr/managers.py (hunk 2 above)
Comment (s-i-l-k-e) [reply python/aristotle-metadata-registry/aristotle_mdr/managers.py]: Refactored

File: python/aristotle-metadata-registry/aristotle_mdr/managers.py (hunk 3 above)
Comment (s-i-l-k-e) [reply python/aristotle-metadata-registry/aristotle_mdr/managers.py]: Decided

**Pull request merged**